Alesab/cosmos

alesab-08 opened the pull request:
I did the computation for \sigma_\theta and added a suggestion about the use of \Delta over \delta.

alesab-08 committed:
Update cosmos.tex

computation of \sigma_\theta and small suggestion

alesab-08 committed:
Update cosmos.tex

shows a line that I don't see in the code

joanalnu commented on COSMOS-Programme/cosmos.tex:
```diff
@@ -93,12 +93,13 @@ \section{Time-Delay localisation and intensity interferometry}\label{sec:time-de
 The key innovation of the COSMOS programme is to leverage the already developed high-energy observing capabilities from heritage missions and introduce arcminute-level source localisation via coordinated observations. This is a known concept in physics, for instance various GW observatories (LIGO, VIRGO and KAGRA) together can further constraint the sky coordinates of a signal than a single one \ ...
 
-The main idea is that comparing the signals from the same source detected by different satellites we can tighter constrain the origin of the source, in terms of celestial coordinates. For two satellites separated at baseline distance $\mathbf{B}$ and the time-delay between both detectors is $\delta t = t_2 - t_1$ then the direction of the source in the sky is given by
+The main idea is that comparing the signals from the same source detected by different satellites we can tighter constrain the origin of the source, in terms of celestial coordinates. For two satellites separated at baseline distance $\mathbf{B}$ and the time-delay between both detectors is $\delta t = t_2 - t_1$ then the direction of the source in the sky is given by %SUGGESTION:use \Delta inste ...
```
```suggestion
The main idea is that comparing the signals from the same source detected by different satellites we can tighter constrain the origin of the source, in terms of celestial coordinates. For two satellites separated at baseline distance $\mathbf{B}$ and the time-delay between both detectors is $\Delta t = t_2 - t_1$ then the direction of the source in the sky is given by 
```

joanalnu replied:
Sure!

joanalnu commented on COSMOS-Programme/cosmos.tex:
```suggestion
	\left(\begin{matrix} \mathbf{B}_{21} \\ \mathbf{B}_{31} \end{matrix} \right) \hat{s} = c \left(\begin{matrix}\Delta t_{21} \\ \Delta t_{31}\end{matrix}\right)
```

joanalnu commented on COSMOS-Programme/cosmos.tex:
```diff
@@ -133,6 +134,24 @@ \section{Time-Delay localisation and intensity interferometry}\label{sec:time-de
+&= \frac{31.6\,\mathrm{m}}{|\mathbf{B}_\perp|}
+\end{align}
+
+We deduce that $\sigma_\theta$ is inversely proportional to $|\mathbf{B}_\perp|$; as the baseline gets larger, the uncertainty decreases. To estimate it, we assign $|\mathbf{B}_\perp|$ some numerical values: $|\mathbf{B}_\perp| = 100\,\mathrm{m}, 250\,\mathrm{m}, 500\,\mathrm{m}, 1000\,\mathrm{m}$.
```
```suggestion
Note that that $\sigma_\theta \sim \frac{1}{\mathbf{B}_\perp$; as the baseline gets larger, the uncertainty decreases.
```

joanalnu commented on COSMOS-Programme/cosmos.tex:
```diff
@@ -133,6 +134,24 @@ \section{Time-Delay localisation and intensity interferometry}\label{sec:time-de
+\sigma_{\theta,3} &= \frac{31.6\,\mathrm{m}}{500\,\mathrm{m}} = 6.32 \times 10^{-2}\,\mathrm{rad} \approx 3.62^\circ \\
+\sigma_{\theta,4} &= \frac{31.6\,\mathrm{m}}{1000\,\mathrm{m}} = 3.16 \times 10^{-2}\,\mathrm{rad} \approx 1.81^\circ
+\end{align}
+
```
```suggestion

```

alesab-08 replied:
Sorry Joan, I just want to know if this has to go because I got it wrong and it wasn't what I was supposed to do or because I have to do it in another way

joanalnu replied:
Hi @alesab-08, you did nothing wrong, but instead of this set of formulas I think it is better to put your results in a table, which I put in another content. Thus the content doesn’t has to go, only the format.

joanalnu reviewed (changes requested):
Good job @alesab-08!

alesab-08 committed:
Update cosmos.tex

Apply requested changes

joanalnu committed:
Apply suggestion from @joanalnu

Signed-off-by: Joan <139717293+joanalnu@users.noreply.github.com>

joanalnu committed:
Apply suggestion from @joanalnu

Signed-off-by: Joan <139717293+joanalnu@users.noreply.github.com>

joanalnu commented on COSMOS-Programme/cosmos.tex:
```diff
@@ -133,6 +134,34 @@ \section{Time-Delay localisation and intensity interferometry}\label{sec:time-de
+Note that $\sigma_\theta \sim \frac{1}{\mathbf{B}_\perp}$; as the baseline gets larger, the uncertainty decreases.
+
+
 \textcolor{red}{add a brief computation for the values from \ref{sec:technical_feasibility_and_heritage} ($\sigma_t = 100 {\ \rm ns}$, $\sigma_{B, ||} \approx 10 {\ \rm m}$)}
```
```suggestion

```

joanalnu commented on COSMOS-Programme/cosmos.tex:
```diff
@@ -133,6 +134,34 @@ \section{Time-Delay localisation and intensity interferometry}\label{sec:time-de
+        3 & 500  & 0.0632 & 3.62 \\
+        4 & 1000 & 0.0316 & 1.81 \\
+        \hline
+    \end{tabular}
```
```suggestion
    \end{tabular}
    \label{tab:angular_uncertainty}
```

joanalnu reviewed (changes requested):
Some changes from the last review weren’t applied (format-wise). Added a couple of comments to sentences to improve flow.

alesab-08 committed:
Apply suggestion from @joanalnu

Co-authored-by: Joan <139717293+joanalnu@users.noreply.github.com>
Signed-off-by: AleSab <ale.sabi2008@gmail.com>

alesab-08 committed:
Apply suggestion from @joanalnu

Co-authored-by: Joan <139717293+joanalnu@users.noreply.github.com>
Signed-off-by: AleSab <ale.sabi2008@gmail.com>

alesab-08 committed:
Apply suggestion from @joanalnu

Co-authored-by: Joan <139717293+joanalnu@users.noreply.github.com>
Signed-off-by: AleSab <ale.sabi2008@gmail.com>

alesab-08 committed:
Apply suggestion from @joanalnu

Co-authored-by: Joan <139717293+joanalnu@users.noreply.github.com>
Signed-off-by: AleSab <ale.sabi2008@gmail.com>

alesab-08 committed:
Apply suggestion from @joanalnu

Co-authored-by: Joan <139717293+joanalnu@users.noreply.github.com>
Signed-off-by: AleSab <ale.sabi2008@gmail.com>

joanalnu commented on COSMOS-Programme/cosmos.tex:
```diff
@@ -133,7 +134,28 @@ \section{Time-Delay localisation and intensity interferometry}\label{sec:time-de
+        \hline
+    \end{tabular}
+    \label{tab:angular_uncertainty}
+    \caption{Angular uncertainty as a function of distance}
```
```suggestion
    \caption{Angular uncertainty as a function of satellite baseline distance}
```

joanalnu commented on COSMOS-Programme/cosmos.tex:
```diff
@@ -133,7 +134,28 @@ \section{Time-Delay localisation and intensity interferometry}\label{sec:time-de
 \end{equation}
 
-\textcolor{red}{add a brief computation for the values from \ref{sec:technical_feasibility_and_heritage} ($\sigma_t = 100 {\ \rm ns}$, $\sigma_{B, ||} \approx 10 {\ \rm m}$)}
+For reference, given $\sigma_t = 100 \ \mathrm{ns}$ and $\sigma_{B, \parallel} \approx 10 \ \mathrm{m}$, we obtain the values in table \ref{tab:angular_uncertainty}.
```
```suggestion
For reference, given $\sigma_t = 100 {\ \mathrm{ns}}$ and $\sigma_{B, \parallel} \approx 10 {\ \mathrm{m}}$, we obtain the values in table \ref{tab:angular_uncertainty}.
```

alesab-08 committed:
Apply suggestion from @joanalnu

Co-authored-by: Joan <139717293+joanalnu@users.noreply.github.com>
Signed-off-by: AleSab <ale.sabi2008@gmail.com>

alesab-08 committed:
Apply suggestion from @joanalnu

Co-authored-by: Joan <139717293+joanalnu@users.noreply.github.com>
Signed-off-by: AleSab <ale.sabi2008@gmail.com>

joanalnu committed:
Apply suggestion from @joanalnu

Signed-off-by: Joan <139717293+joanalnu@users.noreply.github.com>

joanalnu committed:
Fix unit formatting and remove duplicate labels

Corrected formatting for units in the text and removed duplicate label entries in the table.

Signed-off-by: Joan <139717293+joanalnu@users.noreply.github.com>

joanalnu committed:
laTeX formating issues
